backend/app/routes/admin/updateAdmin.js

```diff
@@ -0,0 +1,36 @@
+const to = require('await-to-js').default;
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const userRecord = await Admin.findOne({
+    email: res.locals.decode.email,
+  });
+
+  // Finding user through email which needs to be updated
+  const user = await Admin.findOne({ email: userRecord.email });
```

Comment:
Why do we need this? Please use only 1 DB

Reply:
So here's the issue I'm facing, when I call `Admin.findOneAndUpdate()`, I need to use the user we find  and then make changes. How do I use the `user` before it is even initialized? So for this:

- one call is being made to the db to fetch the user
- one call to make the changes

How do I make both these calls in a single call?

Reply:
I encourage you to deep dive on the solution. There's no point if we spoon feed you the solution.

Reply:
Ok, so I fixed this issue. But due to this, another issue popped up. Whenever I hit the route `updateAdmin.js` , information gets updated real time on the database but in the response, that change is seen after 1 extra click. But database is getting updated real time even though response fails to do so.

When I update:
This is what shows in the response
![image](https://user-images.githubusercontent.com/52799877/110846665-0bf0b700-82d2-11eb-98fb-5b83d9d86b69.png)
(Notice how the contact is the non updated one)

And this is what shows in database
![image](https://user-images.githubusercontent.com/52799877/110846735-232fa480-82d2-11eb-9c43-f2fedda09846.png)

Reply:
Look for the fix :p 
It will help you in the long run